api/settings.py

```diff
@@ -23,8 +23,10 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = config('SECRET_KEY')
 
+
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', default=True, cast=bool)
+#DEBUG = config('DEBUG', default=True, cast=bool)
```

Comment:
can you explain this change

Reply:
Config file was not working on my laptop so I changed those settings, I will take care of removing my settings and restore to previous one in my next pull request.

Reply:
read about `python-decouple` and how it is used in the django projects, you will not need to manage env var

Reply:
Ok I will work on it.